notebooks/HASP/DataDiagnostic/DataDiagnostics.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<p>Same as the other output log above:</p><pre class="ql-syntax" spellcheck="false">with open(logfile, 'r') as f:
  for line in f:
    print(line.strip()) 
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>